### PR TITLE
fix: set default permanentSessionLifetime to 30 days instead of 30 hours

### PR DESCRIPTION
**Pull request by bidord**
Docker image uses a default value for `PERMANENT_SESSION_LIFETIME` of [30\*24\*3600](https://github.com/Mailu/Mailu/blob/f17aa85b54430421253d9e003f42cb3b0d7866ee/core/admin/mailu/configuration.py#L81) (30 days).

Helm chart, on the other hand, uses a default value of `108000` (`=30*3600`, so 30 hours).

I understand that sometimes it makes sense to use different default values for kubernetes. But this particular value of 30 hours (not 24 or 48) makes me think someome just forgot to multiply by 24.

This PR replaces the default value for `permanentSessionLifetime` with the same one as in the docker image: `2592000` (`=30*24*3600`).

### CODE REVIEW

**Comment by fastlorenzo** [comment]
Good catch, thanks for the PR

**Review by fastlorenzo — APPROVED** [review]

File: mailu/values.yaml [hunk 1]
```diff
@@ -204,7 +204,7 @@ sessionCookieSecure: true
 sessionTimeout: 3600
 
 ## @param permanentSessionLifetime Maximum amount of time in seconds a session can be kept alive for if it hasn’t timed-out
-permanentSessionLifetime: 108000
+permanentSessionLifetime: '2592000'
```
Comment (fastlorenzo) [inline mailu/values.yaml:207]: Could you remove the quotes here please?

**Review by fastlorenzo — CHANGES_REQUESTED** [review]

**Comment by fastlorenzo** [comment]
Looks like the check is failing: 

```
Traceback (most recent call last):
  File "/app/venv/bin/flask", line 8, in <module>
    sys.exit(main())
  File "/app/venv/lib/python3.10/site-packages/flask/cli.py", line 1050, in main
    cli.main()
  File "/app/venv/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/app/venv/lib/python3.10/site-packages/click/core.py", line 1651, in invoke
    cmd_name, cmd, args = self.resolve_command(ctx, args)
  File "/app/venv/lib/python3.10/site-packages/click/core.py", line 1698, in resolve_command
    cmd = self.get_command(ctx, cmd_name)
  File "/app/venv/lib/python3.10/site-packages/flask/cli.py", line 578, in get_command
    app = info.load_app()
  File "/app/venv/lib/python3.10/site-packages/flask/cli.py", line 308, in load_app
    app = locate_app(import_name, name)
  File "/app/venv/lib/python3.10/site-packages/flask/cli.py", line 235, in locate_app
    return find_best_app(module)
  File "/app/venv/lib/python3.10/site-packages/flask/cli.py", line 63, in find_best_app
    app = app_factory()
  File "/app/mailu/__init__.py", line 109, in create_app
    return create_app_from_config(config)
  File "/app/mailu/__init__.py", line 39, in create_app_from_config
    config.init_app(app)
  File "/app/mailu/configuration.py", line 153, in init_app
    self.config['PERMANENT_SESSION_LIFETIME'] = int(self.config['PERMANENT_SESSION_LIFETIME'])
```

Needs some debugging before it can be merged

**Comment by bidord** [comment]
Sorry, I should have mentioned it. I had put the quotes there for a reason. Without them the value is converted to scientific notation by Helm:

```
$ helm template . --set=domain=test.local | grep PERMANENT_SESSION_LIFETIME
  PERMANENT_SESSION_LIFETIME: "2.592e+06"
```

Under the hood, what seems to happen is that YAML parses all numbers as float64 (even small ones). Then, if it is big enough, Helm's `quote` function would convert it to scientific notation:

In `envvars-configmap.yaml`:

```yaml
  PERMANENT_SESSION_LIFETIME: {{ .Values.permanentSessionLifetime | quote }}
  DEBUG_TYPE_OF_PERMANENT_SESSION_LIFETIME: {{ .Values.permanentSessionLifetime | typeOf }}
```

```
$ helm template . --set=domain=test.local | grep PERMANENT_SESSION_LIFETIME
  PERMANENT_SESSION_LIFETIME: "2.592e+06"
  DEBUG_TYPE_OF_PERMANENT_SESSION_LIFETIME: float64
```
Although adding quotes does resolve this problem, I agree it is not ideal. Users who want to change this value would have to put the quotes as well in there custom `values.yaml` or end up with the same problem.

A better alternative would probably be something like my last push:

In `envvars-configmap.yaml`:

```yaml
  PERMANENT_SESSION_LIFETIME: {{ .Values.permanentSessionLifetime | int64 | quote }}
```

```
$ helm template . --set=domain=test.local | grep PERMANENT_SESSION_LIFETIME
  PERMANENT_SESSION_LIFETIME: "2592000"
```

What do you think?

**Pull request merged**